.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: mastercomfig Discord
+    url: https://discord.gg/CuPb2zV
+    about: Discord server used for Team Comtress questions and discussion
```

Comment:
dot?

Reply:
It's a sentence fragment, so idk if it needs one